Code/Editor/EditorPreferencesPageGeneral.cpp

```diff
@@ -76,6 +77,8 @@ void CEditorPreferencesPage_General::Reflect(AZ::SerializeContext& serialize)
             ->DataElement(AZ::Edit::UIHandlers::CheckBox, &GeneralSettings::m_previewPanel, "Show Geometry Preview Panel", "Show Geometry Preview Panel")
             ->DataElement(AZ::Edit::UIHandlers::CheckBox, &GeneralSettings::m_applyConfigSpec, "Hide objects by config spec", "Hide objects by config spec")
             ->DataElement(AZ::Edit::UIHandlers::CheckBox, &GeneralSettings::m_enableSourceControl, "Enable Source Control", "Enable Source Control")
+            ->DataElement(
+                AZ::Edit::UIHandlers::CheckBox, &GeneralSettings::m_clearConsoleOnGameModeStart, "Clear Console at Game Startup", "Clear Console when Game Mode Starts")
```

Comment:
We don't use Initcaps. inside o3de. We only cap proper nouns and the first word. This should read "Clear Console when game mode starts." "Game mode" is a tricky phrase. I would say that it's not a proper name but could be convinced otherwise. Other than that the location and control look great

Reply:
Done, thanks for the feedback.